javaparser-core/src/main/java/com/github/javaparser/printer/lexicalpreservation/Difference.java

```diff
@@ -485,78 +537,80 @@ void apply(NodeText nodeText, Node node) {
         }
         boolean addedIndentation = false;
         List<TokenTextElement> indentation = LexicalPreservingPrinter.findIndentation(node);
+
+        List<TextElement> givenElements = nodeText.getElements();
```

Comment:
Given is not super-clear to me as a term in this context

Reply:
of course 🤦‍♂️ thanks